### ISSUE TITLE
sendMulticast() successfully sends notifications but they never arrive

### ISSUE THREAD

**Issue by ctissier** [kreait/firebase-php#414]
**The bug**
I've an issue with the `sendMulticast()` method. I want to send a notification to three different devices, 1 Android and 2 iOS.

When I send the notification to one device at a time with the `send()` method, the notification arrives immediately on the choosen device.

But when I group the notifications with the `sendMulticast()` method, the notifications never arrive.

**To Reproduce**
Here's what I do:

```php
$tokens = ['abc....','def.....'];

if(!empty($tokens)){
    $message = Messaging\CloudMessage::new()
                    ->withNotification(Notification::create($title,$content));

    $report = $this->firebaseMessaging->sendMulticast($message, $tokens);

    echo 'Successful sends: '.$report->successes()->count().PHP_EOL;
    echo 'Failed sends: '.$report->failures()->count().PHP_EOL;  
 }
```

The result of this call is:

```
Successful sends: 3   
Failed sends: 0
```

But no device recieves the notification...

Did I miss something somewhere in the configuration process or in the `sendMulticast()` call?

**Expected behavior**
I expect the call of `sendMulticast()` would send in one batch the same notification to several devices.

**Developping Environment (please complete the following information):**
 - OS: MacOS 10.15.4
 - PHP version: 7.3.11
 - Symfony version: 4.4
 - kreait/clock version: 1.0.1
 - kreait/firebase-bundle version: 2.0.0 
 - kreait/firebase-php version: 5.1.1
 - kreait/firebase-tokens version: 1.10.0 

**Additional context**
I didn't try on the `staging` or `prod` environment. But as it works when sending the notifications individually, I presume the multicast send should work as well.

**Comment by jeromegamez** [comment]
You didn't miss anything, and that the Firebase APIs returned 3 successes for the 3 messages indicates that the message didn't contain errors and the three targets were valid.

The difference between `send()` and `sendMulticast()` from the SDK perspective is that instead of sending one request to the `https://fcm.googleapis.com/v1/projects/<project-id>/messages:send` endpoint, three individual requests are "concatenated" and sent as one request to the `https://fcm.googleapis.com/batch` endpoint (I hope the explanation makes sense, I'm not quite sure myself at the moment :)).

Long story short, I believe this is a (hopefully temporary) delivery issue on the FCM side, from the SDK side there is not much I can do 😞

(Just to be clear, I would prefer it to be an SDK bug, because then I could fix it 😅)

**Comment by ctissier** [comment]
That totally makes sense... it so makes sense that you were right: I just resent the notification and the three devices received the notification instantly.

So the issue was indeed on the FCM side and not on the SDK side.

Thank you very much for your really quick answer and keep on the good work.

**Comment by jeromegamez** [comment]
Thank you, and you're welcome 🌺. Glad it was just a glitch in the FCmatrix (it's bad-puns-wednesday 😅)
